src/angular/components/sso.component.ts

```diff
@@ -61,7 +69,7 @@ export class SsoComponent {
                 this.clientId = qParams.clientId;
             }
 
-            if (qParams.clientId === ConstantsService.browserClientId) {
+            if (qParams.clientId == ConstantsService.browserClientId) {
```

Comment:
Why not use the explicit equality test? Was there a reason perhaps `===` was failing the test while the equalish test was passing?